tests/unit/setup.ts

```diff
@@ -28,3 +29,13 @@ Vue.use(TiptapVuetifyPlugin, {
   // optional, default to 'md' (default vuetify icons before v2.0.0)
   iconsGroup: 'mdi'
 })
+
+// Mock feature flags for unit tests.
+jest.spyOn(util, 'GetFeatureFlag').mockImplementation(
+  (name) => {
+    if (name === 'supported-alteration-change-business-types') {
```

Comment:
If this is too global, let me know, I can copy and paste it into the 3-4 different sections or call a function instead

Reply:
This is too global -- other features may want different feature flags.

Nice code, though :)

Reply:
adding it to the list for next PR